eng/pipelines/mono/templates/build-job.yml

```diff
@@ -53,8 +53,10 @@ jobs:
       value: ${{ parameters.osSubgroup }}
     - name: officialBuildIdArg
       value: ''
-    - name: osOverride
-      value: ''
+    # Strip symbols only on the release build
+    - ${{ if eq(parameters.buildConfig, 'Release') }}:
+      - name: stripSymbolsArg
+        value: '-stripsymbols'
```

Comment:
I don't think this argument is supported anymore. I believe @am11 removed this. 

Also, why are you removing the `osOverride`?